src/averageAges.js

```diff
@@ -37,7 +46,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenList = people.filter(person => withChildren
+    ? people.find(child => person.name === child.mother)
```

Comment:
you can use some function instead of find here. Find is usually used when you want to find an item and use it somewhere else